contract_sale_invoicing/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=======================
+Contract Sale Invoicing
+=======================
+
+This module extends the functionality of contract adding the possibility
```

Comment:
This modules allows to include on the same invoice than the recurring invoice contract, all the pending to invoice sales orders that you have with the same analytic account.